libraries/AP_HAL_ChibiOS/hwdef/GEPRCF745BTHD/README.md

```diff
@@ -0,0 +1,97 @@
+# GEPRC TAKER F745 BT Flight Controller
+
+The TAKER F745 BT is a flight controller produced by [GEPRC](https://geprc.com/).
+
+## Features
+
+ - STM32F745 microcontroller
+ - MPU6000+ICM42688 dual IMU
+ - BMP280 barometer
+ - microSD based 512MB flash logging
+ - AT7456E OSD
+ - 7 UARTs
+ - 8 PWM outputs
+
+## Pinout
+
+![TAKER F745 BT Board](TAKER_F745_BT_Board_Top.jpg "GEPRCF745BTHD")
+![TAKER F745 BT Board](TAKER_F745_BT_Board_Bottom.jpg "GEPRCF745BTHD")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (DJI-VTX, DMA-enabled)
+ - SERIAL2 -> UART2 (RCIN, DMA-enabled) 
+ - SERIAL3 -> UART3 (connected to internal BT module, not currently usable by ArduPilot)
+ - SERIAL4 -> UART4 (GPS)
+ - SERIAL6 -> UART6 (leisure)
+ - SERIAL7 -> UART7 (leisure)
+ - SERIAL8 -> UART8 (ESC Telemetry)
+
+## RC Input
+
+RC input is configured on the R2 (UART2_RX) pin. It supports all RC protocols.
+ 
+## FrSky Telemetry
+ 
+FrSky Telemetry is supported using the Tx pin of any UART including SERIAL2/UART2. You need to set the following parameters to enable support for FrSky S.PORT (example shows SERIAL3).
+ 
+  - SERIAL3_PROTOCOL 10
+  - SERIAL3_OPTIONS 7
+  
+## OSD Support
+
+The TAKER F745 BT supports OSD using OSD_TYPE 1 (MAX7456 driver).
+
+## VTX Support
+
+The SH1.0-6P connector supports a standard DJI HD VTX connection. Pin 1 of the connector is 12v so be careful not to connect
```

Comment:
```suggestion
The SH1.0-6P connector supports a standard DJI HD VTX connection. Pin 1 of the connector is 12v (or VBAT by solder pad selection)so be careful not to connect
```

Reply:
I dont know what the Chinese says for the VTX pin...please translate...also what is the difference between the VSW and this pin? do you have a power switch onboard? its not included in the hwdef...